.github/workflows/reusable-build-test-release.yml

```diff
@@ -290,8 +290,7 @@ jobs:
       - uses: actions/setup-python@v5
       - name: remove dev dependencies
         run: | 
-          sudo apt install crudini
-          pip install poetry==1.5.1
+          pip3 install crudini poetry==1.5.1
```

Comment:
Let's have a PR with the logic - let's not run fossa if there is no main dependencies.

I'll talk to Andrzej and we will get an approval for that from legal.

Reply:
@artemrys I reworked the PR. I didn't handle `THIRDPARTY` file as we are waiting for the decision from legal team